task/sbom-json-check/0.1/sbom-json-check.yaml

```diff
@@ -41,40 +43,76 @@ spec:
       source /utils.sh
       trap 'handle_error $(results.TEST_OUTPUT.path)' EXIT
 
+      images_processed_template='{"image": {"pullspec": "'"$IMAGE_URL"'", "digests": [%s]}}'
       mkdir /manifests/ && cd /manifests/
 
-      image_with_digest="${IMAGE_URL}@${IMAGE_DIGEST}"
+      imagewithouttag=$(echo $IMAGE_URL | sed "s/\(.*\):.*/\1/" | tr -d '\n')
+      image_with_digest=$(echo $imagewithouttag@$IMAGE_DIGEST)
+      digests_processed=()
 
-      if ! oc image extract --registry-config ~/.docker/config.json "${image_with_digest}" --path '/root/buildinfo/content_manifests/*:/manifests/'; then
-        echo "Failed to extract manifests from image ${image_with_digest}."
-        note="Task $(context.task.name) failed: Failed to extract manifests from image ${image_with_digest} with oc extract. For details, check Tekton task log."
-        ERROR_OUTPUT=$(make_result_json -r "ERROR" -t "$note")
+      # check if the image is a manifest list
```

Comment:
nit: this comment isn't necessary anymore since `get_image_manifests` now fetches the image even if it isn't a manifest list.